SwiftCompilerSources/Sources/SIL/Instruction.swift

```diff
@@ -356,10 +362,14 @@ final public class GlobalValueInst : GlobalAccessInst {}
 final public class IntegerLiteralInst : SingleValueInstruction {}
 
 final public class TupleInst : SingleValueInstruction {
+  public var uniqueNonTrivialElt: Value? {
```

Comment:
This is actually a utility and not a property of the instruction. Please move this to `ReleaseDevirtualizer.swift` (I don't expect it to be used by any other pass).
You can implement that in swift, no need to bridge it.